modules/renter/persist.go

```diff
@@ -4,94 +4,54 @@ import (
 	"bytes"
 	"compress/gzip"
 	"encoding/base64"
-	"encoding/json"
 	"errors"
 	"io"
 	"os"
 	"path/filepath"
 	"strconv"
+
+	"github.com/NebulousLabs/Sia/persist"
 )
 
 const (
 	PersistFilename = "renter.dat"
```

Comment:
looks like some of the filenames are *.json, and others are *.dat

We should be consistent